### PR TITLE
[designate-nanny] moving user seeds to designate

### PR DESCRIPTION
**Pull request by mutax**
To enable seeding in all namespaces necessary, user seeding has moved to the designate helm chart, to continue having a single source of truth the nanny charts now read the designate values as well.

Enabling the nanny and the users used is now configured in value files of designate centrally.

### CODE REVIEW

File: openstack/designate/templates/seeds.yaml [hunk 1]
```diff
@@ -938,3 +938,42 @@ spec:
       role: cloud_dns_admin
       inherited: true
 {{- end }}
+
+{{- if .Values.nanny_enabled }}
+---
+apiVersion: "openstack.stable.sap.cc/v1"
+kind: "OpenstackSeed"
+metadata:
+  name: designate-nanny-seed
```
Comment (galkindmitrii) [inline openstack/designate/templates/seeds.yaml:947]: would you like to replace designate here with {{ .Release.Name }} ?
In that case seeds for global case will be called different, automatically.

File: openstack/designate/values.yaml [hunk 2]
```diff
@@ -7,6 +7,7 @@ debug: "True"
 
 global_setup: false
 tempest_enabled: false
+nanny_enabled: false
```
Comment (galkindmitrii) [inline openstack/designate/values.yaml:10]: since this is coming to all region, I think it is fine to leave it to true per default.

**Review by galkindmitrii — APPROVED** [review]

File: openstack/designate/values.yaml (hunk 2 above)
Comment (mutax) [reply openstack/designate/values.yaml:10]: I just aligned it with tempest and had the idea to have a "safe" default in the helm charts to be overwritten where we need it. in global for example we don't have it in all regions and when its true by default there will be missing configuration.

File: openstack/designate/templates/seeds.yaml (hunk 1 above)
Comment (mutax) [reply openstack/designate/templates/seeds.yaml:947]: oh, makes sense. the name then will be designate-global-nanny which is not in sync with the general naming scheme of designate-nanny-global.  But will change it, makes sense!

**Review by galkindmitrii — APPROVED** [review]

**Pull request merged**